src/components/common/CookieAndTermBanner/index.tsx

```diff
@@ -72,8 +74,11 @@ export const CookieBanner = ({
         <Grid container alignItems="center">
           <Grid item xs>
             <Typography variant="body2" mb={2}>
-              By clicking &quot;Accept all&quot; you agree to the use of the tools listed below and their corresponding
-              cookies. <ExternalLink href={AppRoutes.cookie}>Cookie policy</ExternalLink>
+              By browsing this page, you accept our{' '}
+              <ExternalLink href={AppRoutes.terms}>Terms & Conditions</ExternalLink> and the use of necessary cookies.
+              By clicking &quot;Accept al &quot; or &quot;Save settings &quot;, you additionally agree to the use of
```

Comment:
I feel like this sounds as if "Accept all" and "Save settings" would both accept all options.
I was confused by that text and would drop the Save settings part:
`By clicking "Accept all" you additionally agree to the use of Beamer and Analytics cookies as listed below.`